job/server/src/main/java/alluxio/master/job/JobMaster.java

```diff
@@ -91,6 +93,17 @@ public WorkerNetAddress getFieldValue(MasterWorkerInfo o) {
         }
       };
 
+  /**
+   * The Filesystem context that the job master uses for its client.
+   */
+  private final FileSystemContext mFsContext =
+      FileSystemContext.create(ServerConfiguration.global());
+
+  /**
+   * The FileSystem client the job master uses to select executors for jobs.
+   */
+  private final FileSystem mFileSystem = FileSystem.Factory.create(mFsContext);
```

Comment:
Any reason for now doing this in the constructor similar to `JobWorker`?

Reply:
I just modified JobMaster so that the client/context are passed in the same way as the JobWorker